toonz/sources/include/saveloadqsettings.h

```diff
@@ -0,0 +1,18 @@
+#pragma once
+
+#ifndef SAVE_LOAD_QSETTINGS_INCLUDED
+#define SAVE_LOAD_QSETTINGS_INCLUDED
+
+#include <QSettings>
+
+class QSettings;
+
+//! An interface that claims: this object wants to save / load something
+//! into / from provided qsettings
+class SaveLoadQSettings {
```

Comment:
**[THIS IS NOT THE CHANGE REQUEST!]**
This implementation is NICE ! It's better to use this class for some existing parameters instead of using user .env file. Thanks!  :+1: